tests/e2e/specs/cli/index.test.js

```diff
@@ -1,38 +1,29 @@
 const assert = require("assert");
 const path = require("path");
-const tmp = require("tmp");
 const fs = require("fs");
 const exec = require("child_process").exec;
 import CLI from "@src/cli";
 const timeout = 5e4;
 const quire = new CLI();
-
-// static names and paths for quire test
-const sandboxDir = tmp.dirSync({
-  unsafeCleanup: true
-});
-const projectName = "quire-starter";
-const themeName = "default";
-const CONFIG = {
-  STARTER_THEME: themeName,
-  DEFAULT_PROJECT_NAME: projectName,
-  PROJECT_FOLDER: path.join(sandboxDir.name, projectName),
-  STATIC_FILES_PATH: path.join(
-    sandboxDir.name,
-    projectName,
-    "static",
-    "downloads"
-  ),
-  THEME_PATH: path.join(sandboxDir.name, projectName, "themes", themeName)
-};
+import CONFIG from "../../fixtures/project-config.js";
 
 describe("CLI", () => {
-  process.chdir(sandboxDir.name);
+  test(
+    "test project exists",
+    async (done) => {
+      await assert.equal(fs.existsSync(path.join(CONFIG.FIXTURES_DIR, CONFIG.PROJECT_FOLDER)), true);
+
+      process.chdir(path.join(CONFIG.FIXTURES_DIR, CONFIG.PROJECT_FOLDER));
+
+      done();
+    },
+    timeout
+  );
```

Comment:
I think this could be written more clearly as
```suggestion
  test(
    "test project exists", 
    async (done) => {
      const projectPath = path.join(CONFIG.FIXTURES_DIR, CONFIG.PROJECT_FOLDER);
      await assert(fs.existsSync(projectPath)).toBe(true);
      process.chdir(projectPath);
      done();
    },
    timeout
  );
```
though in practice writing unit tests that make only a single assertion are easier to read and maintain; this test calls the `process.chdir` command after the test assertion, which should be moved to a separate test and perhaps `const projectPath = path.join(CONFIG.FIXTURES_DIR, CONFIG.PROJECT_FOLDER);` should be moved to a `beforeAll` statement.